python/fusion_engine_client/messages/configuration.py

```diff
@@ -399,3 +414,164 @@ def __str__(self):
+        }
+
+
+class SetOutputInterfaceStreamsMessage(MessagePayload):
+    """!
+    @brief Configure the set of output streams enabled for a given output interface.
+    """
+    MESSAGE_TYPE = MessageType.SET_OUTPUT_INFERFACE_STREAMS
+    MESSAGE_VERSION = 0
+
+    SetOutputInterfaceStreamsMessageConstruct = Struct(
+        "update_action" / AutoEnum(Int8ul, UpdateAction),
+        Padding(3),
+        "output_interface_config" / _OutputInterfaceConfigConstruct(),
+    )
+
+    def __init__(self):
+        self.update_action = UpdateAction.REPLACE
+        self.output_interface_config = OutputInterfaceConfig()
+
+    def pack(self, buffer: bytes = None, offset: int = 0, return_buffer: bool = True) -> (bytes, int):
+        packed_data = self.SetOutputInterfaceStreamsMessageConstruct.build(self.__dict__)
+        return PackedDataToBuffer(packed_data, buffer, offset, return_buffer)
+
+    def unpack(self, buffer: bytes, offset: int = 0) -> int:
+        parsed = self.SetOutputInterfaceStreamsMessageConstruct.parse(buffer[offset:])
+        self.__dict__.update(parsed)
+        return parsed._io.tell()
+
+    def __str__(self):
+        fields = ['update_action', 'output_interface_config']
+        string = f'Set Output Interface Streams Command\n'
+        for field in fields:
+            val = str(self.__dict__[field]).replace('Container:', '')
+            string += f'  {field}: {val}\n'
+        return string.rstrip()
+
+    def calcsize(self) -> int:
+        return len(self.pack())
+
+
+class GetOutputInterfaceStreamsMessage(MessagePayload):
+    """!
+    @brief Query the value of the output interface stream configuration.
+    """
+    MESSAGE_TYPE = MessageType.GET_OUTPUT_INFERFACE_STREAMS
+    MESSAGE_VERSION = 0
+
+    GetOutputInterfaceStreamsMessageConstruct = Struct(
+        "request_source" / AutoEnum(Int8ul, ConfigurationSource),
+        Padding(3),
+    )
+
+    def __init__(self):
+        self.request_source = ConfigurationSource.ACTIVE
+
+    def pack(self, buffer: bytes = None, offset: int = 0, return_buffer: bool = True) -> (bytes, int):
+        packed_data = self.GetOutputInterfaceStreamsMessageConstruct.build(self.__dict__)
+        return PackedDataToBuffer(packed_data, buffer, offset, return_buffer)
+
+    def unpack(self, buffer: bytes, offset: int = 0) -> int:
+        parsed = self.GetOutputInterfaceStreamsMessageConstruct.parse(buffer[offset:])
+        self.__dict__.update(parsed)
+        return parsed._io.tell()
+
+    def __str__(self):
+        fields = ['request_source']
+        string = f'Get Config Command\n'
+        for field in fields:
+            val = str(self.__dict__[field]).replace('Container:', '')
+            string += f'  {field}: {val}\n'
+        return string.rstrip()
+
+    @classmethod
+    def calcsize(cls) -> int:
+        return cls.GetOutputInterfaceStreamsMessageConstruct.sizeof()
+
+
+class OutputInterfaceStreamsDataMessage(MessagePayload):
```

Comment:
I'm not in love with these names. Technically they do say what is going on, but it feels like having both words in the name actually makes things more confusing (and more of a mouthful). Maybe just `Get/SetOutputInterfaceConfig` and `OutputInterfaceConfigResponse`?

Reply:
We've already gone back and forth on "Response" for the queried data previously. I think it makes sense since we're adding response fields, but at some point we need to just stick with a consistent set of names.